capstone-frontend/src/LeftSideBar.test.js

```diff
@@ -13,8 +13,14 @@ import {
 const container = document.createElement("div");
 document.body.appendChild(container);
 
+const getUserBookLists = () => {
+  return [{ id: "1", name: "Best Books" },
```

Comment:
nit: Break this line after the `[` then indent the objects in the array (https://google.github.io/styleguide/jsguide.html#formatting-array-literals).

Reply:
Ok